### DISCUSSION TITLE
Is it possible to disable github runner autoupdate?

### DISCUSSION THREAD

**Discussion by stefangluszek** [actions/actions-runner-controller#1356] [Questions] [answered]
Is it possible to disable the [autoupdate](https://github.blog/changelog/2022-02-01-github-actions-self-hosted-runners-can-now-disable-automatic-updates/) of the GH runners?

**Comment by toast-gear** [comment, marked as answer]
https://github.com/actions-runner-controller/actions-runner-controller#runner-entrypoint-features

**Reply by stefangluszek** [reply]
Awesome, thanks!
